multiversx_sdk_core/utils.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
```

Comment:
File can be renamed to `io.py` or something similar (opinion).

Reply:
Renamed to `io.py`